src/components/Layout/SideNav/index.scss

```diff
@@ -1,5 +1,11 @@
 @import '~scss/vars';
 
+%disabledLink {
+  color: $N75;
+  opacity: 0.5;
+  cursor: not-allowed;
+}
+
 .nav {
```

Comment:
Suggest this style can remove _mixins.scss file, if do this other file can reference.

Reply:
ok